instrumentation/github.com/astaxie/beego/common.go

```diff
@@ -0,0 +1,50 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package beego
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/otel/api/kv"
+)
+
+// ------------------------------------------ Constants
+
+const (
+	// packageName is the name of the this package, and is used as the default tracer
+	// and meter names.
+	packageName = "go.opentelemetry.io/contrib/instrumentation/github.com/astaxie/beego"
+
+	renderTemplateSpanName = "beego.render.template"
+	renderStringSpanName   = "beego.render.string"
+	renderBytesSpanName    = "beego.render.bytes"
+
+	templateKey = kv.Key("go.template")
```

Comment:
Should this key be prefixed with `beego`?

```suggestion
	templateKey = kv.Key("beego.go.template")
```

Reply:
I was following the example of the gin-gonic instrumentation which uses [`go.template`](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/97f04f73f9f669f1d99492f6b914154149497037/instrumentation/github.com/gin-gonic/gin/gintrace.go#L105).

Beego uses the [html/template](https://golang.org/pkg/html/template/) package, so I thought that it made more sense to leave it as `go.template`. Happy to change it if you think otherwise!

Reply:
Ah, makes sense. Leaving sounds good. :+1: